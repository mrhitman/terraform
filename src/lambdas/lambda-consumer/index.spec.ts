import { handler } from './';

describe('lambda emitter', () => {
  describe('happy path', () => {
    test('success', async () => {
      const event = {} as any;
      const response: any = await handler(event, null as any, null as any);
      expect(response).toMatchSnapshot();
    });

    test('success2', async () => {
      const event = {} as any;
      const response: any = await handler(event, null as any, null as any);
      expect(response).toMatchSnapshot();
    });
  });
});
