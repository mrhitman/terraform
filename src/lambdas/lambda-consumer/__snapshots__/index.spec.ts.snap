// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lambda emitter happy path success 1`] = `
Object {
  "body": "{
  \\"message\\": \\"lambda 1 assadasdas \\",
  \\"input\\": {}
}",
  "statusCode": 200,
}
`;

exports[`lambda emitter happy path success2 1`] = `
Object {
  "body": "{
  \\"message\\": \\"lambda 1 assadasdas \\",
  \\"input\\": {}
}",
  "statusCode": 200,
}
`;
